lib/expand/join.go

```diff
@@ -821,12 +819,12 @@ func (p *Peer) validateWizardState(operator ops.Operator) (*ops.Site, *ops.SiteO
 	}
 
 	switch operation.State {
-	case ops.OperationStateInstallInitiated, ops.OperationStateFailed:
+	case ops.OperationStateInstallInitiated, ops.OperationStateInstallProvisioning, ops.OperationStateFailed:
 		// Consider these states for resuming the installation
 		// (including failed that puts the operation into manual mode)
 	default:
-		return nil, nil,
-			trace.AlreadyExists("operation %v is in progress", operation)
+		return nil, nil, trace.AlreadyExists("operation %#v is in progress",
```

Comment:
Why do you want to dump the object directly in the error message? Why not use `log.Warnf()`

Reply:
It was already being dumped here, I just added detailed format.